code-for-canada/project-thundercat
# Remove dialog to enter instructions

# Description

Remove dialog to enter instructions. We talked with the team about how this was redundant information. I'm working on dialog accessibility (and other accessibility work) and having a dialog to start is blocking us from testing other elements of the simulation.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

# Testing

Manual steps to reproduce this functionality:

1.  Go to the sample test page.
2.  Enter the emib and notice no dialog before getting to the instructions.

# Checklist

Applicable for all code changes.
